testsuite/tests/properties/logging/test.py

```diff
@@ -62,5 +62,5 @@ class Ref(FooNode):
     ref=Ref(fg.name),
     name=Name(Token.Identifier),
 )
-build_and_run(fg, 'main.py')
+build_and_run(fg, ada_main='main.adb')
```

Comment:
Could you also remove `main.py`? Thanks!

Reply:
Argh sorry, yes

Reply:
Thanks! :-)